README.md

```diff
@@ -96,6 +97,10 @@ At `_controller` you define controller and action for every document type.
 
 `urlAnalyzer` at `ongr_elasticsearch` configuration defines how all url fields are analyzed by Elasticsearch.
 
+If you are using another third party bundle that also has a predefined chain router, you may set `disable_alias` to `true`. This
```

Comment:
It's not necessary to mention chained router below, it's about Symfony router not chained.